src/core/IronPython.Modules/ModuleOps.cs

```diff
@@ -357,13 +354,8 @@ public static IntPtr GetObject(object value) {
         }
 
         public static long GetSignedLongLong(object value, object type) {
-            int? res = Converter.ImplicitConvertToInt32(value);
-            if (res != null) {
-                return res.Value;
-            }
-
-            if (value is BigInteger) {
-                return (long)(BigInteger)value;
+            if (PythonOps.TryToInt(value, out BigInteger bi)) {
+                return unchecked((long)(ulong)(bi & ulong.MaxValue));
```

Comment:
I guess this will no longer throw if the value is too big for a `long`. Not too familiar with these methods, but just confirming that this is the intended behaviour?

Reply:
I guess `test_conversions_overflow` answers my question...